docs/topics/native/native-cocoapods-libraries.md

```diff
@@ -285,4 +285,29 @@ kotlin {
         }
     }
 }
+```
+
+### Dependencies between Pods
+
+You can use the cinterop binding generated for another Pod when building a binding for the new Pod.
+The dependent Pod should be declared before setting up the dependency. The Kotlin CocoaPods Gradle plugin supports two ways
+of adding dependencies between Pods:
+
+* `useInteropBindingFrom()` function specifies the name of the existing Pod that is used as dependency.
+* `interopBindingDependencies` contains a list of all dependencies to other Pods.
```

Comment:
I guess we can omit `interopBindingDependencies` from the docs. It was added just for consistency. Most of the users should be fine with `useInteropBindingFrom`